seckill-command-service/src/main/java/io/servicecomb/poc/demo/seckill/web/SecKillAdminController.java

```diff
@@ -0,0 +1,21 @@
+/*
+ *   Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package io.servicecomb.poc.demo.seckill.web;
+
+public interface SecKillAdminController {
+  String create(PromotionCreate create);
```

Comment:
it's better to move admin stuff to a separate module and start up as an individual service, to avoid interference by high volume of command service